### PR TITLE
Docs: add more information about graph QL and PWA integration fro Magento 2

### PR DESCRIPTION
**Pull request by Stanislau-MultiSafepay**

### CODE REVIEW

File: content/integrations/plugins/magento2/faq/pwa-support.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Can I integrate the plugin in my PWA storefront?"
+meta_title: "Magento 2 plugin FAQ - PWA integration - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+read_more: "."
+---
+
+Yes, plugin fully compatible with GraphQL queries and it's easy possible to integrate into your PWA store with our additional separate [module for GraphQL support](https://github.com/MultiSafepay/magento2-graphql)
```
Comment (Vinod-MultiSafepay) [inline content/integrations/plugins/magento2/faq/pwa-support.md:8]:
```
Yes, the plugin is fully compatible with GraphQL queries and it is possible to easily integrate it into your PWA store with an additional separate [module for GraphQL support](https://github.com/MultiSafepay/magento2-graphql)
```

File: content/integrations/plugins/magento2/manual.md [hunk 2]
```diff
@@ -35,10 +34,11 @@ Some of the new features include:
 * Clear explanation of each payment method with links to docs
 * Modular setup, providing more flexibility during the installation process
 * For some payment methods the hosted payment page is now skipped, resulting in a higher conversion rate
+* PWA (GraphQL) endpoints
 
 ### 2. Requirements
 - To use the plugin you need a MultiSafepay account. You can create a test account on https://testmerchant.multisafepay.com/signup
-- Magento Open Source version 2.2.x & 2.3.x & 2.4.x
+- Magento Open Source version 2.2.x & 2.3.x & 2.4.x (For GraphQL support Magento Open Source version 2.4.x)
```
Comment (Vinod-MultiSafepay) [inline content/integrations/plugins/magento2/manual.md:41]: - Magento Open Source version 2.2.x & 2.3.x & 2.4.x (For GraphQL only Magento Open Source version 2.4.x is supported)

File: content/integrations/pwa/pwa-studio/_index.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+download_url : "https://github.com/MultiSafepay/magento2-graphql.git"
 changelog_url : "."
 manual: "." 
 layout: 'single'
-meta_title: "PWA Studio plugin integration - MultiSafepay Docs"		
-meta_description: "MultiSafepay plugin for PWA Studio. Easily integrate MultiSafepay payment solutions into your PWA Studio platform with the free plugin"
-description : "Easily integrate MultiSafepay payment solutions into your PWA Studio webshop with the free and completely new MultiSafepay PWA Studio plugin."
+meta_title: "PWA for plugin integration - MultiSafepay Docs"		
+meta_description: "MultiSafepay plugin for PWA integration. Easily integrate MultiSafepay payment solutions into your PWA based store with the free plugin"
+description : "Easily integrate MultiSafepay payment solutions into your PWA webshop with the free and completely new MultiSafepay plugin for GraphQL queries support"
```
Comment (Vinod-MultiSafepay) [inline content/integrations/pwa/pwa-studio/_index.md:11]: I would ditch the word 'support' and say something like:

`description : "Easily integrate MultiSafepay payment solutions into your PWA webshop with the free and completely new MultiSafepay plugin for GraphQL queries and mutations"`

or

`description : "Easily integrate MultiSafepay payment solutions into your PWA webshop with the free and completely new MultiSafepay plugin for GraphQL"`

File: content/integrations/pwa/pwa-studio/_index.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 weight: 130
 logo: "/logo/Plugins/Magento_PWA.svg"
-title_short: "PWA Studio"
-description_short: "Easily integrate MultiSafepay payment solutions into your PWA Studio webshop with the free plugin."
+title_short: "PWA Studio,  ScandiPWA and other"
```
Comment (Vinod-MultiSafepay) [inline content/integrations/pwa/pwa-studio/_index.md:14]: Maybe change it to either one of these:

`title_short: "PWA Studio,  ScandiPWA and others"`

`title_short: "PWA Studio,  ScandiPWA and more"`

File: content/integrations/pwa/pwa-studio/manual.md [hunk 5]
```diff
@@ -1,9 +1,13 @@
 ---
-title : "MultiSafepay PWA Studio"
-meta_title: "PWA Studio plugin manual - MultiSafepay Docs"
+title : "MultiSafepay PWA Integration"
+meta_title: "PWA store integration plugin manual - MultiSafepay Docs"
 meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
 ---
 
-### Using PWA Studio
+### Integrate into PWA store using a separate extension for GraphQL support
```
Comment (Vinod-MultiSafepay) [inline content/integrations/pwa/pwa-studio/manual.md:7]: I would say something like this (up to you to change it or not):

`### Integrate into PWA store using a separate extension that adds GraphQL support`

File: content/integrations/pwa/pwa-studio/manual.md [hunk 6]
```diff
@@ -1,9 +1,13 @@
...
 
-Please see our GitHub repo for the [installation instructions](https://github.com/MultiSafepay/Magento2Msp/tree/progressive-web-app) for PWA Studio.
+Please see our GitHub repo for the [integration instruction and GraphQL request examples](https://github.com/MultiSafepay/magento2-graphql) for PWA based stores.
```
Comment (Vinod-MultiSafepay) [inline content/integrations/pwa/pwa-studio/manual.md:9]: `Please see our GitHub repo for the [integration instructions and GraphQL request examples](https://github.com/MultiSafepay/magento2-graphql) for PWA based stores.`

File: content/integrations/pwa/pwa-studio/manual.md [hunk 7]
```diff
@@ -1,9 +1,13 @@
...
+
+### PWA theme payment component
+
+Please, keep in mind that you will be need create a custom payment component for PWA storefront, depends from which you are using.
```
Comment (Vinod-MultiSafepay) [inline content/integrations/pwa/pwa-studio/manual.md:13]: Please keep in mind that you will need to create a custom payment component for PWA storefront, depending on which PWA storefront you are using.

**Review by Vinod-MultiSafepay — CHANGES_REQUESTED** [review]
I sugggested some improvements. Maybe it is also a good idea to let Kris review it. He has even better english knowledge then i have.

File: content/integrations/plugins/magento2/faq/pwa-support.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+---
+title : "Can I integrate the plugin in my PWA storefront?"
+meta_title: "Magento 2 plugin FAQ - PWA integration - MultiSafepay Docs"
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
+read_more: "."
+---
+
+Yes, the plugin is fully compatible with GraphQL queries and it is possible to easily integrate it into your PWA store with an additional separate [module for GraphQL support](https://github.com/MultiSafepay/magento2-graphql)
```
Comment (Kris-MultiSafepay) [inline content/integrations/plugins/magento2/faq/pwa-support.md:8]:
```suggestion
Yes, the plugin is compatible with GraphQL queries and easy to integrate into your PWA store with an additional [module for GraphQL support](https://github.com/MultiSafepay/magento2-graphql)
```

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]

**Pull request merged**